doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,130 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+Lots of GitHub issues and pull requests are being created daily.
+To help us manage them we have “labels”:
+to classify them by area, type, priority, or to mark them in need of attention.
+
+All GitHub issues or pull requests must be appropriately labeled.
+Issues and PRs often have multiple labels assigned,
+to help classify them in the different available categories.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, reduces the changes of duplicates, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``Area:*``
+Applicable to  PRs  and issues
+Description    Indicates which areas are affected by the bug or pull request
```

Comment:
I thought I had added a comment about this, but doesn't look like I saved it.  Instead of using the word "area" to define what an "area" is, how about something like:
```
Indicates subsystems (e.g., Kernel, I2C, Memory Management),
project functions (e.g., Debugging, Documentation, Process), or other
categories (e.g., Coding Style, MISRA-C)  affected by the bug or pull request.
```

Reply:
Done :)